### PR TITLE
add support for Victoria Metrics Single

### PR DESCRIPTION
**Pull request by Nokel81**
Signed-off-by: Jari Kolehmainen <jari.kolehmainen@gmail.com>

recreating https://github.com/lensapp/lens/pull/4693 as I accidentally merged it before it was ready

### CODE REVIEW

**Review by ixrock — DISMISSED** [review]

File: src/main/prometheus/victoria-metrics-single.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
```
Comment (nevalla) [inline src/main/prometheus/victoria-metrics-single.ts:4]: Please rebase or update license header to the latest version to avoid future lint issues.

**Review by aleksfront — APPROVED** [review]

**Review by DmitriyNoa — APPROVED** [review]

**Comment by jakolehm** [comment]
https://github.com/lensapp/lens/pull/4693 was not ready and probably this is exactly in the same state.. @Nokel81 should we  mark this as a draft?

**Comment by Nokel81** [comment]
@jakolehm yes good point.

**Comment by Nokel81** [comment]
Closing as I don't have time to fix this.

**Pull request closed without merging**